app/views/pages/get_started.html.erb

```diff
@@ -29,7 +29,7 @@
 
 <p>You’ll need your government email address to create a trial account.</p>
 
-<%= govuk_start_button text: "Create a trial account", href: Settings.forms_admin.base_url %>
+<%= govuk_start_button text: "Create a trial account", href: "#{Settings.forms_admin.base_url}/sign-up" %>
```

Comment:
Minor suggestion, maybe create this as a route  so something like `\sign-up` which redirects the user to forms admin signup or as a setting on its own so we dont have to hardcode this link in other places in the app

Reply:
I think that would add a lot of complexity that might end up being unnecessary. @jamie-o-wilkinson has a really handy rule of thumb about <abbr title="Don't Repeat Yourself">DRY</abbr> vs <abbr title="You Aren't Gonna Need it">YAGNI</abbr>, he talks about how repeating two or even three times can be okay. I'm not sure where he got that from but I've found it useful to consider.

As we're only using this URL in one place currently, I think it's better if we leave it hardcoded until we actually do have